### PR TITLE
Add arc proposal real withdraw transaction

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: blockchain/txvalidator.go [hunk 1]
```diff
@@ -273,12 +273,17 @@ func (b *BlockChain) CheckTransactionContext(blockHeight uint32,
 		}
 		return nil
 
+	case CRCProposalRealWithdraw:
+		if err := b.checkCRCProposalRealWithdrawTransaction(txn, references); err != nil {
+			log.Warn("[checkCRAssetsRectifyTransaction],", err)
```
Comment (Houshoupei84) [inline blockchain/txvalidator.go:278]: log.warn should be  "checkCRCProposalRealWithdrawTransaction" err

Comment (RainFallsSilent) [reply blockchain/txvalidator.go]: fixed

File: blockchain/blockchain.go [hunk 2]
```diff
@@ -481,17 +470,37 @@ func (b *BlockChain) CreateCRCAppropriationTransaction() (*Transaction, error) {
...
+		return nil, err
+	}
+
+	payload := &payload.CRCProposalRealWithdraw{
+		WithdrawTransactionHashes: withdrawTransactionHashes,
+	}
+
+	var tx *Transaction
+	tx, err = b.createTransaction(payload, CRCProposalRealWithdraw,
+		b.chainParams.CRCFoundation, Fixed64(RectifyTxFee), uint32(0), utxos, outputs...)
```
Comment (xiaomingfuckeasylife) [inline blockchain/blockchain.go:497]: using RealWithdrawSingleFee instead of RectifyTxFee

Comment (RainFallsSilent) [reply blockchain/blockchain.go:497]: fixed

**Pull request merged**